libp2p/pubsub/gossipsub.py

```diff
@@ -75,14 +73,11 @@ def __init__(
         self.fanout = {}
 
         # Create peer --> protocol mapping
-        self.peers_to_protocol = {}
+        self.peer_protocol = {}
```

Comment:
Are we assuming each peer only supports one protocol here?

Reply:
Yes, a peer supports either floodsub or gossipsub.